components/WelcomeScreen.tsx

```diff
@@ -16,21 +18,25 @@ const WelcomeScreen = ({ handleSignup }: Props): JSX.Element => (
         Here, you can view documentation for and gain access to the{' '}
         {process.env.API_NAME}.
       </p>
-      {isApiManagerEnabled() && (
-        <p>
-          <Button className="mr-3" onClick={handleSignup} variant="primary">
-            Sign up for API access
-          </Button>
-          {process.env.API_DOCUMENTATION_URL && (
-            <Button
-              href={process.env.API_DOCUMENTATION_URL}
-              variant="outline-primary"
-            >
-              View API documentation
+      {
+        /* If we're specifying a USAGE_ID then we definitely don't want to
+        be creating API keys in that AWS environment! */
+        isApiManagerEnabled() && USAGE_ID === null && (
```

Comment:
Should this be enabling account creation when USAGE_ID is null? Shouldn't it be the opposite?

Reply:
If USAGE_ID is set, then we don't have full access to the AWS instance, so probably shouldn't be creating AWS users and keys! If it is UNset, then we are safe to do so